src/test/integration/suite/deployment-logs.test.it.ts

```diff
@@ -0,0 +1,151 @@
+import { EnvironmentModel } from "../../../get-environments";
+import { mockGitRepoAndBranch } from "../mocks/git";
+import {
+  getOutputChannelLogs,
+  mockOutputChannel,
+  outputChannelMock,
+  resetOutputChannelMocks,
+} from "../mocks/output-channel";
+import {
+  mockDeploymentLogsResponses,
+  mockGetEnvironment,
+  mockGetOrganization,
+} from "../mocks/server";
+import {
+  clickOnEnvironmentByName,
+  clickOnFirstEnvironment,
+  getEnvironmentMock,
+  getFirstEnvStatus,
+  login,
+  logout,
+  redeploy,
+  resetExtension,
+  waitFor,
+} from "./test-utils";
+import { afterEach, beforeEach } from "mocha";
+import expect from "expect";
+import { DeploymentStatus } from "../../../types";
+
+const auth = { keyId: "key-id", secret: "key-secret" };
+const orgId = "org-id";
+const firstEnvName = "my env";
+const secondEnvName = "my env 2";
+
+let firstEnvironmentMock: EnvironmentModel;
+let secondEnvironmentMock: EnvironmentModel;
+
+suite("deployment logs", function () {
+  this.timeout(1000 * 15);
```

Comment:
what is this timeout?

Reply:
this is the test timeout
3bfa17983c1e965e1addb62032c2981cbfdad006
now its on 10 sec as for all the tests, the reason is that some times it takes a while until the `setInterval` updates the state